Shared/DsaController.cpp

```diff
@@ -607,6 +630,59 @@ bool writeJsonFile(QIODevice& device, const QSettings::SettingsMap& map)
   return writtenBytes != -1;
 }
 
+Viewpoint viewpointFromJson(QJsonObject initialLocation)
```

Comment:
can this method be `const`? or even `static`?

Reply:
This is a freestanding function not in a class method context.

Reply:
D'oh! I see. Does it need to be in the header or can we put it in an anonymous namespace? If not, how you have it is fine

Reply:
Because `writeJsonFile` and `readJsonFile` weren't, I was following convention. But if there's no convention there I'll wrap them all up!